### PR TITLE
File Name Correction

### PR DESCRIPTION
**Pull request by harshit-paneri**
`docker-compose.yaml` -> `docker-compose-core.yaml`

### CODE REVIEW

**Comment by harshit-paneri** [comment]
Line Change : 

Expose the port in port for otel-collector in `docker-compose.yaml `file present in `deploy/docker/clickhouse-setup` to Expose the port in port for otel-collector in `docker-compose-core-config.yaml `file present in `deploy/docker/clickhouse-setup`

PR #911

**Comment by harshit-paneri** [comment]
Review :  @nityanandagohain , @Calm-Rock

File: data/docs/userguide/fluentd_to_signoz.mdx [hunk 1]
```diff
@@ -72,7 +72,7 @@ At SigNoz we use opentelemetry collector to recieve logs which supports the flue
 
 ## Collect Logs Using FluentD in Self-Hosted SigNoz
 ### Steps to recieve logs from FluentD:
-* Add fluentforward reciever to your `otel-collector-config.yaml` which is present inside `deploy/docker/clickhouse-setup`
+* Add fluentforward reciever to your `docker-compose-core.yaml` which is present inside `deploy/docker/clickhouse-setup`
```
Comment (nityanandagohain) [inline data/docs/userguide/fluentd_to_signoz.mdx]: this should be `otel-collector-config.yaml`

File: data/docs/userguide/fluentd_to_signoz.mdx [hunk 2]
```diff
@@ -109,7 +109,7 @@ At SigNoz we use opentelemetry collector to recieve logs which supports the flue
     ```
     Here we are adding our clickhouse exporter and creating a pipeline which will collect logs from `fluentforward` receiver, processing it using batch processor and export it to clickhouse.
 
-* Expose the port in port for otel-collector in `docker-compose.yaml` file present in `deploy/docker/clickhouse-setup`
+* Expose the port in port for otel-collector in `otel-collector-config.yaml` file present in `deploy/docker/clickhouse-setup`
```
Comment (nityanandagohain) [inline data/docs/userguide/fluentd_to_signoz.mdx]: this should be `docker-compose-core.yaml`

**Review by nityanandagohain — CHANGES_REQUESTED** [review]

**Review by harshit-paneri — COMMENTED** [review]
New code, Check it

**Review by nityanandagohain — APPROVED** [review]

**Comment by Calm-Rock** [comment]
Thanks for the PR @harshit-paneri. Merged!

**Pull request merged**